### PR TITLE
post: optimising Spark DB perf

### PR DESCRIPTION
**Pull request by bajtos**
Preview: https://deploy-preview-21--bajtos.netlify.app/posts/2023-11-optimising-spark-db-perf/

### CODE REVIEW

File: content/posts/2023-11-optimising-spark-db-perf.md [hunk 1]
```diff
@@ -0,0 +1,371 @@
...
+title: Optimising performance of Spark's Postgres database
+date: 2023-11-30
+---
+
+On November 13th, we publicly launched Filecoin Station, a desktop app enabling
+everybody to participate in the Filecoin economy and earn FIL for contributing
+their spare computing resources & network bandwidth. (You can download the app
+here: https://www.filstation.app). The launch was a success, and our network
+grew from ~50 to more than ~1500 nodes in a few days. We quickly discovered that
+our database has a hard time keeping up with the increased load.
```
Comment (juliangruber) [inline content/posts/2023-11-optimising-spark-db-perf.md]: idea: mention the current network size, to give a sense for scale?

File: content/posts/2023-11-optimising-spark-db-perf.md [hunk 2]
```diff
@@ -0,0 +1,371 @@
...
+
+## Metrics
```
Comment (juliangruber) [inline content/posts/2023-11-optimising-spark-db-perf.md:28]: would be nice to visualize this section with some screenshots

File: content/posts/2023-11-optimising-spark-db-perf.md [hunk 3]
```diff
@@ -0,0 +1,371 @@
...
+Important: By default, Cloudflare enforces a minimal `max-age` value of 4 hours
+for cache-able content. I had to turn off this behaviour in the caching
+configuration by asking Cloudflare to honor `max-age` returned by the server.
+
+## Grafana Dashboards
+
+The changes I made in our backend significantly improved the performance:
+
+- CPU load decreased to 0.4
+- Max query duration went down to 0.1 ms
```
Comment (juliangruber) [inline content/posts/2023-11-optimising-spark-db-perf.md]: would be nice to give context: which were the values before your optimization?

File: content/posts/2023-11-optimising-spark-db-perf.md (hunk 2 above)
Comment (bajtos) [reply content/posts/2023-11-optimising-spark-db-perf.md:28]: Yeah, I considered this too but then decided to keep the post text-only.

**Pull request merged**